client/src/components/WarmFuzziesGiven.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import UserClient from "../apiClient/UserClient";
+import BackWarmFuzzy from "./BackWarmFuzzy";
+import { useAuthentication } from "../providers/Authentication";
+
+const WarmFuzziesGiven = (props) => {
+  const { user } = useAuthentication();
+  const [warmFuzzies, setWarmFuzzies] = useState([]);
+
+  const fetchWarmFuzzies = () => {
+    const client = new UserClient()
+    const allWarmFuzzies = client.getGivenFuzzies(user.userId)
+      .then(resp => {
+        return resp;
+      })
+      .then(fuzzies => {
+        setWarmFuzzies(fuzzies)
+      })
+  }
+
+  useEffect(fetchWarmFuzzies, []);
+
+  const allFuzzies = warmFuzzies.map((fuzzy => {
+    let showAllEmail;
+    return (
+      <BackWarmFuzzy
```

Comment:
I would actually pass a `fuzzy` prop instead of all the individual properties.

So instead of `title={fuzzy.title}` etc, just use a `fuzzy={fuzzy}` prop and handle the object accordingly in the `BackWarmFuzzy` component.